src/app/components/Hero/Hero.tsx

```diff
@@ -6,20 +6,16 @@ import { HeroInfo } from "./HeroInfo";
 export const Hero = () => {
   return (
     <section id="hero">
+      <div className="block">
+        <HeroImg />
```

Comment:
On some resolutions from 710px to 834px and 1290px to 1439px the image goes into the icons and texts. I think it would be a good idea to add custom media just for this component (don't make them in variables anymore like tablet and desktop). Unless you find another better solution :)

Reply:
I looked through the code and it seems to me that the reason for the overlapping image on the icons and other page elements, is the use of relative units. In this project we don't have a smooth responsive design, but three breakpoints with rigid values. As such, the use of vw units causes the fully responsive element to “collide” with the rest of the page, which does not have this responsiveness. 

Still, it's a good idea to add an extra scope for the Image component for proper image rendering
( sizes="(max-width: 834px) 305px, (min-width: 835px) 480px; (min-width: 1440px) 733px” - just check if I gave the correct values ;) )